tsconfig.json

```diff
@@ -9,7 +9,6 @@
     "target": "es2017"
   },
   "include": [
-    "src/",
-    "tests/"
+    "src/"
```

Comment:
Deletion of `tests/` is the fix. It had been added in the v1.0.0 PR (#23) to ensure that `tests/` files were type checked, but it led to the incorrect behaviour of subfolders being created under `build/`:
```
build/src/index.js
build/src/index.js.map
build/tests/...
```
It should instead look like:
```
build/index.js
build/index.js.map
```
The correct behaviour is achieved by having a single folder listed in the `include` array.

As for type checking the tests files, it is done by `gulp` and `ts-node` anyway:

**package.json**
```
"test": "gulp test",
```
**gulpfile.js**
```
function test() {
	return gulp.src('tests/tests.ts').pipe(
		gmocha({
			require: ['ts-node/register'],
		}),
	);
}
```